### ISSUE TITLE
AM x Screen Daily advertisement

### ISSUE THREAD

**Issue by cams-goussale** [blockframes/blockframes#8212]
@Lorena-Amaral needs 3 ads to promote Archipel Market. They will be published in Screen Daily's Product Guide during Cannes. A Product Guide is a book distributed to all Buyers and Sellers during Cannes, which holds all the names and all the movie line ups of the Sellers.

In this book, we'll have 3 full pages to promote AM.

Specs:
- It will be printed on paper
- Detailed specs can be found here: [screenproductguidespecs_10498.pdf](https://github.com/blockframes/blockframes/files/8450859/screenproductguidespecs_10498.pdf)
- Design 3 full pages including the content below + space for a QR code + illu
- We can use the graphic chart of Cannes, with gold colors

### 1st page
=> Explain what the partnership is about
H1: `Let's go digital!`
Logos AM + Logos Screen
H2: `Digitalize your Cannes experience. Check out Sellers' line-ups on Archipel Market by scanning the QR code below.`
QR Code
H3: `At Cannes 2022, Screen and Archipel Market are partnering to offer an innovative and more powerful Product Guide. This means twice as much visibility for Cannes participants — and access to the 1,500 worldwide members of Archipel Market.`

### 2nd page
=> Explain what AM is for
We could reuse this and change Meetings for Privacy Settings
QR Code

![image](https://user-images.githubusercontent.com/93206363/162419890-7372d946-7c4d-42ed-838c-7b50e5c95d54.png)

### 3rd page
=> Invitation to meet us and to have a demo
`Come meet us at Cannes!`
CTA: `Schedule a demo with us`
QR Code

We can also reuse this image somewhere, as we find it catchy. We could add here "For Free".
![image](https://user-images.githubusercontent.com/93206363/162419615-017ef9a1-00a2-45c4-b993-4abb61df4144.png)

**Comment by cams-goussale** [comment]
Ideally, it'd be nice to have the QR code on each page, but it really depends on the structure of each page. We could try to include a tagline next to the QR code like:

`Come discover Screen's Cannes Product Guide on Archipel Market!`

**Comment by cams-goussale** [comment]
Please @afleutot @Lorena-Amaral make sure you discuss with Stéphanie when she works on this.

**Comment by Lorena-Amaral** [comment]
If the (archipelmarket.com) URL doesn't change for the mobile version, here follows the QR code:
![frame](https://user-images.githubusercontent.com/101707820/162441019-4792c9be-4e8c-4ac9-8b19-d08f21fd8bb0.png)

**Comment by cams-goussale** [comment]
```
At Cannes 2022, Screen and Archipel Market are partnering to offer an innovative and more powerful Product Guide. This 2022 version is not only printed! Sales slates are also on the Archipel Market platform. This means twice as much visibility for Cannes participants — and access to the 1,500 worldwide members of Archipel Market.
[Archipelmarket.com](http://archipelmarket.com/) is the leading ongoing film market platform for promoting line-ups and organising free screenings.
```

**Comment by cams-goussale** [comment]
[Drive links to upload png](https://drive.google.com/drive/u/0/folders/12Z4W2ChvZoUyvpKAmq25xtj06J9zNudf)

**Comment by afleutot** [comment]
Idée pour 3ème page:
![3rd page](https://user-images.githubusercontent.com/99681152/163203196-3a1fc901-2ba5-4161-8f69-981fe3118d81.png)

Let's go Digital...
...though come meet us here in Cannes for a demo!
Contact Lorena Amaral at: lamaral@cascade8.com
QR Code

**Comment by stephanie-cascade8** [comment]
salut @afleutot @Lorena-Amaral voici une première base discussion. 
Du coup j'ai fait deux modes: 
Un jour (Cannes / soleil...) et un nuit (plutôt Cannes by night), à voir. On se fait un point pour échanger? OU vous pouvez m'envoyer des demandes de changements / essais...
![Cannes-By-Night (1)](https://user-images.githubusercontent.com/66258851/163992295-d8c929bc-1e42-4673-8862-830a780c8e65.png)
![Cannes-In-the-Sun](https://user-images.githubusercontent.com/66258851/163992326-a1f80245-f5bd-478e-a1a9-eb6b0e14bc70.png)
![Cannes-In-the-Sun](https://user-images.githubusercontent.com/66258851/163992397-48a3518d-96ab-4b06-ae99-32d4cea313b6.png)

**Comment by afleutot** [comment]
Hello Stéphanie, je trouve ça sublime! Je demande à Screen leur logo en vectorisé ou fichier original et je t'envoie leur réponse. @Lorena-Amaral je te laisse regarder pour qu'on puisse caler un point avec Stéphanie?

**Comment by afleutot** [comment]
![screen](https://user-images.githubusercontent.com/99681152/164023312-daeb1bb0-5328-4315-99de-af328e43ba5d.png)
Et voilà @stephanie-cascade8 ! Tiens moi au courant si tu souhaites que je te l'envoie autrement ;)

**Comment by stephanie-cascade8** [comment]
> ![screen](https://user-images.githubusercontent.com/99681152/164023312-daeb1bb0-5328-4315-99de-af328e43ba5d.png) Et voilà @stephanie-cascade8 ! Tiens moi au courant si tu souhaites que je te l'envoie autrement ;) Je l'ai également placé dans le drive à Logos partenaires!

Super Merci!  Je mets le logo à jour et continue sur cette lancée pour les autres pages. De votre côté dites moi quand vous êtes dispo pour faire des ajustement ensemble ;)

**Comment by stephanie-cascade8** [comment]
Hi @Lorena-Amaral @afleutot ! 
I added some changes following our last discussion. 
I had some new thoughts that will be up to you : 

- I think having the contact of a person in particular on a printed add seems weird. We usually put the contact@archipelmarket.com... (otherwise I think it makes it look like their is only on person behind the service, don't you think?)

- On the 3rd screen, to illustrate the idea of the opposition btw digital & physical Cannes market, I think we need to have the dots only once in between. Maybe something like the repetition of "let's" helps enhancing this contrast we want to express. Tell me if you have some news on your side about this sentence.

- What do you think of having a minimalist 3rd page to close the trilogy? 
I'll let you set the appointment with Yannick for the next steps (I'll be available for this task in priority next week).

PS: sorry about the space before the exclamation point in the first page. (the text is a drawing for now for technical purposes so I'll correct this in the end after all changes).

<img width="400" alt="Capture d’écran 2022-04-21 à 18 59 41" src="https://user-images.githubusercontent.com/66258851/164512924-dff38b11-ba89-47be-a2d7-9898205cbe28.png">

**Comment by cams-goussale** [comment]
@Lorena-Amaral @afleutot I think the "Privacy settings" is not very explicit for external people. My advice is to rephrase it, to express what we mean by that. These privacy settings allow users to "Share their films in a secure way" or to "Invite whoever they want with just 1 link". I don't have the right words in mind, but imo Privacy settings is not clear.

**Comment by Lorena-Amaral** [comment]
> @Lorena-Amaral @afleutot I think the "Privacy settings" is not very explicit for external people. My advice is to rephrase it, to express what we mean by that. These privacy settings allow users to "Share their films in a secure way" or to "Invite whoever they want with just 1 link". I don't have the right words in mind, but imo Privacy settings is not clear.

Thank you Camille, you are right! @afleutot could you please propose a new phrase? I thought of: "Secure invitations"?
